src/Infrastructure/LeanCode.Firebase.FCM/EntityFramework/PushNotificationTokenStore.cs

```diff
@@ -14,6 +14,8 @@ namespace LeanCode.Firebase.FCM.EntityFramework
     public sealed class PushNotificationTokenStore<TDbContext> : IPushNotificationTokenStore
         where TDbContext : DbContext
     {
+        private const int MaxBatchSize = 100;
```

Comment:
Might be a good idea to make it `public` (and rename to `GetTokensMaxBatchSize` for clarity), because it's kind of part of the public contract right now.

Reply:
I disagree - the exact number is not part of the real _contract_. Exposing it just makes it the liability and it won't be used much - most of the time the code would depend on `IPushNotificationTokenStore` than this.

But... it might be a good idea to put in the interface and make it contractual there, so no matter the implementation, we can't expect it to accept more params.